cocos/ui/UIWidget.cpp

```diff
@@ -558,6 +558,10 @@ void Widget::setTouchEnabled(bool enable)
     if (_touchEnabled)
     {
         _touchListener = EventListenerTouchOneByOne::create();
+        if (!_touchListener) {
```

Comment:
When will the `create` method return nullptr? I think it's rare case. Could you provide a test case to verify it?